client/components/account-balances/index.tsx

```diff
@@ -3,8 +3,7 @@
  */
 import * as React from 'react';
 import { Flex, TabPanel } from '@wordpress/components';
-import { __, sprintf } from '@wordpress/i18n';
-import interpolateComponents from '@automattic/interpolate-components';
+import { __ } from '@wordpress/i18n';
```

Comment:
Maybe we can remove this, not being used.

Reply:
Good catch! Removed here https://github.com/Automattic/woocommerce-payments/pull/8140/commits/dcd7b5f2db3c6786ceed5904c167db43bd0b2878.